dialect_sqlite.go

```diff
@@ -88,6 +89,10 @@ func (d SqliteDialect) TruncateClause() string {
 	return "delete from"
 }
 
+func (d SqliteDialect) SleepClause(s time.Duration) string {
+	return ""
+}
```

Comment:
Same here as with the OracleDialect - you wouldn't have to implement this method here.